src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -928,6 +965,12 @@ internal void OnSolutionAnalyzerReferenceRemoved(AnalyzerReference analyzerRefer
         }, WorkspaceChangeKind.SolutionChanged);
     }
 
+    /// <summary>
+    /// Call this method when <see cref="Solution.FallbackAnalyzerOptions"/> change in the host environment.
+    /// </summary>
+    internal void OnSolutionFallbackAnalyzerOptionsChanged(ImmutableDictionary<string, StructuredAnalyzerConfigOptions> options)
```

Comment:
does this need to be internal?

Reply:
Doesn't actually need to exist at all. But we have a pattern of OnXxxChanged methods...

Reply:
i would be totally ok with you removing this, and inlining if only at one callsite.  if we don't need it, at least please make privat.e keeping the surface area from growing is super helpful for thr workspace :)

Reply:
make privat eif possible.